media/js/firefox/new/scene2.js

```diff
@@ -2,55 +2,29 @@
  * License, v. 2.0. If a copy of the MPL was not distributed with this
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
-(function($, dataLayer) {
+(function($) {
     'use strict';
 
     var isIELT9 = window.Mozilla.Client.platform === 'windows' && /MSIE\s[1-8]\./.test(navigator.userAgent);
     var $directDownloadLink = $('#direct-download-link');
     var $platformLink = $('#download-button-wrapper-desktop .download-list li:visible .download-link');
-    var $stage = $('#stage');
-
-    // build virtualUrl for GA
-    var pathParts = window.location.pathname.split('/');
-    var queryStr = window.location.search ? window.location.search + '&' : '?';
-    var referrer = pathParts[pathParts.length - 2];
-    var locale = pathParts[1];
-    var virtualUrl = ('/' + locale + '/products/download.html' +
-                       queryStr + 'referrer=' + referrer);
-
+    var downloadURL;
 
     if ($platformLink.length) {
-        // Pull download link from the download button and add to the
-        // 'click here' link.
+        downloadURL = $platformLink.attr('href');
+
+        // Pull download link from the download button and add to the 'click here' link.
         // TODO: Remove and generate link in bedrock.
-        $directDownloadLink.attr('href', $platformLink.attr('href'));
+        $directDownloadLink.attr('href', downloadURL);
+
+        // if user is not on an IE that blocks JS triggered downloads, start the
+        // platform-detected download after window (read: images) have loaded.
+        // only auto-start the download if a visible platform link is detected.
+        if (!isIELT9 && $platformLink.length) {
```

Comment:
Since we're verifying the truthiness of `$platformLink.length` above, could this check just be `if (!isIELT9)`?